src/domain/space/components/subspaces/CreateSubspaceForm.tsx

```diff
@@ -139,26 +151,21 @@ export const CreateSubspaceForm = ({
           />
           <Gutters padding={theme => `${gutters()(theme)} 0 0 0`}>
             <PageContentBlock sx={{ flexDirection: isSmallScreen ? 'column' : 'row', justifyContent: 'space-between' }}>
-              <FormikVisualUpload name="visuals.avatar" visualType={VisualType.Avatar} flex={1} />
-              <FormikVisualUpload name="visuals.cardBanner" visualType={VisualType.Card} flex={1} />
+              <FormikVisualUpload
+                name="visuals.avatar"
+                visualType={VisualType.Avatar}
+                flex={1}
+                initialVisualUrl={templateVisuals.avatar}
```

Comment:
I don't see the reason to have a `showInitialVisual`, is there any reason to not have just the string, and if the string is undefined or empty interpret it as if `showInitialVisual` was `false`?